lab-raziyeh/lib/fs-files.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function(files){
+    var dataArray = [];
+    fs.readFile(`${__dirname}/../data/1.txt`, function(err, data){
+        if(err) throw err;
+        dataArray.push(data.toString('hex',0,8));
+
+        fs.readFile(`${__dirname}/../data/2.txt`, function(err,data){
+            if(err) throw err;
+            dataArray.push(data.toString('hex',0,8));
+            
+            fs.readFile(`${__dirname}/../data/3.txt`, function(err,data){
+                if(err) throw err;
+                dataArray.push(data.toString('hex',0,8));
+                files(dataArray);
+            });
+        });
+    });
+};
+
```

Comment:
@Raziyehbazargan Indentation seems to be set to 4 spaces per tab - you'll want to update this in your editor settings so that it is always 2 spaces per tab.  Your linter should also be complaining as we have it setup to do so if the spacing is incorrect.

Outside of that, I wish I could nitpick this more and give you some additional pointers but the code here looks excellent!  Outside of the indentation, this file is clean, efficient, and pretty much exactly what we are looking for.  Great job!